Extensions/messaging_tweaks.js

```diff
@@ -366,6 +372,9 @@ XKit.extensions.messaging_tweaks = new Object({
 		if (XKit.extensions.messaging_tweaks.preferences.hide_send_post.value) {
 			XKit.tools.add_css(".messaging-share-post-search, .messaging-share-post-main {display:none;}", "messaging_tweaks");
 		}
+		if (XKit.extensions.messaging_tweaks.preferences.hide_online_indicators.value) {
+			$(".status-indicator-wrapper, .chat-status-banner").hide();
```

Comment:
On closer inspection I think this may need to be done with CSS -- since `.hide()` just hits whatever happens to be in the DOM at the time the extension first runs, it doesn't intercept the new DOM elements when the inbox dropdown is created.